### DISCUSSION TITLE
希望增加视频播放界面的【排版重构功能】和【鼠标滑轮的应用场景拓展】

### DISCUSSION THREAD

**Discussion by amazingand** [the1812/Bilibili-Evolved#4697] [功能建议 / Ideas]
目前B站视频播放界面两侧存在大段白边，导致不论是默认的网页模式还是网页宽屏模式的观感都显得局促。
希望能够仿照油管的视频播放界面：
1、去掉两侧白边；
2、放大默认网页播放窗口；
其次，当前B站网页全屏或全屏状态下不能直接查看视频简介、推荐视频以及评论区。
同样可以学习油管的模式：
1、引入剧场模式代替网页全屏或与之共存；
2、加入全屏后可通过鼠标滚轮滑动查看视频简介、推荐视频以及评论区的功能。

**Comment by Geeyun-JY3** [comment]
这里有个比你的想法更进一步的，作者有回复，可参考：#4180

**Reply by amazingand** [reply]
我在一个油猴脚本上发现有类似的实现：https://greasyfork.org/zh-CN/scripts/445241-mr%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9%E5%8A%A9%E6%89%8B-%E8%87%AA%E5%8A%A8%E5%AE%BD%E5%B1%8F%E6%A8%A1%E5%BC%8F-%E8%87%AA%E5%AE%9A%E4%B9%89%E5%B8%83%E5%B1%80-%E6%99%BA%E8%83%BD%E8%BF%9E%E6%92%AD-%E6%89%93%E9%80%A0%E5%B1%9E%E4%BA%8E%E8%87%AA%E5%B7%B1%E7%9A%84b%E7%AB%99

**Reply by Geeyun-JY3** [reply]
哇试用了一下，这脚本有点东西啊。
